.github/workflows/notebook_ci.yaml

```
@@ -107,15 +102,19 @@ jobs:
         name: executed-notebooks-v${{ matrix.qutip-version }}
         path: |
           notebooks/*.ipynb
-          notebooks/**/*.ipynb
-   
-    - uses: gaurav-nelson/github-action-markdown-link-check@v1
-      with:
-        use-quiet-mode: 'yes'
-        folder-path: tutorials-v${{ matrix.qutip-version }}
-        #use config file to define 403 and 405 errors as valid links
-        #(APS blocks this link check)
-        config-file: mlc_config.json
+          notebooks/**/*.ipynbo
+
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: tcort/github-action-markdown-link-check@v1
+        with:
+          use-quiet-mode: 'yes'
+          folder-path: 'tutorials-v4, tutorials-v4'
```

```suggestion
          folder-path: 'tutorials-v4, tutorials-v5'
```
Same suggestion here.